brands/office/estate_agent.json

```
@@ -104,6 +104,15 @@
       "short_name": "Hamptons"
     }
   },
+  "office/estate_agent|John L. Scott": {
+    "countryCodes": ["us"],
+    "tags": {
+      "brand": "John L. Scott",
+      "brand:wikidata": "Q6243687",
+      "brand:wikipedia": "en:John_L._Scott",
```

Need to switch underscores with spaces, then good to merge 👍 

```suggestion
      "brand:wikipedia": "en:John L. Scott",
```